### PR TITLE
[14.0][IMP] stock_request + stock_request_submit: Show error in stock.request.order in the "Confirm" or "Submit" buttons if there are no items.

### PR DESCRIPTION
**Pull request by victoralmau**
Changes done:
- [x] Show error in `stock.request.order` in the "_Confirm_" or "_Submit_" buttons if there are no items.
- [x] Show Confirm button from `stock.request.order` only to Stock Request Manager users.

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT39761

### CODE REVIEW

File: stock_request/models/stock_request_order.py [hunk 1]
```diff
@@ -223,6 +223,8 @@ def change_childs(self):
                 line.procurement_group_id = self.procurement_group_id
 
     def action_confirm(self):
+        if not self.stock_request_ids:
+            raise UserError(_("Some item must be exist"))
```
Comment (pedrobaeza) [inline stock_request/models/stock_request_order.py]:
```suggestion
            raise UserError(_("There should be at least one request item for confirming the order."))
```

File: stock_request_submit/i18n/es.po [hunk 2]
```diff
@@ -37,6 +56,16 @@ msgstr "Pedido de solicitud de stock"
 msgid "Submit"
 msgstr "Enviar"
 
+#. module: stock_request_submit
+#: code:addons/stock_request_submit/models/stock_request_order.py:0
+#, python-format
+msgid "There should be at least one request item for submiting the order."
+msgstr "Debe haber al menos un elemento de solicitud para enviar el pedido."
+
+#, python-format
```
Comment (pedrobaeza) [inline stock_request_submit/i18n/es.po]: Remove this

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]

**Review by LudLaf — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**